src/xml/namespace.cr

```diff
@@ -4,8 +4,8 @@ struct XML::Namespace
   def initialize(@document : Node, @ns : LibXML::NS*)
   end
 
-  def hash
-    object_id
+  def hash(hasher)
+    hasher << object_id
```

Comment:
Here you use `hasher << object_id` where in `XML::Node#hash` and `XML::NodeSet#hash` you use `hasher.raw object_id`, why is it different?